e2e/cypress/integration/messaging/strikethrough_spec.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Edit post with "strikethrough', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18710-Edit post with "strikethrough" and ensure channel auto-complete closes after second tilde (~~)', () => {
+        const message = 'Hello' + Date.now();
+
+        // # Create new DM channel with user's email
+        cy.apiGetUsers(['user-1', 'sysadmin']).then((userResponse) => {
+            const userEmailArray = [userResponse.body[1].id, userResponse.body[0].id];
+
+            cy.apiCreateDirectChannel(userEmailArray).then(() => {
+                cy.visit('/ad-1/messages/@sysadmin');
+
+                // # Post message to use
+                cy.postMessage(message);
+
+                // # Hit up arrow to open edit modal
+                cy.get('#post_textbox', {timeout: TIMEOUTS.LARGE}).clear().type('{uparrow}');
+
+                // # Type first tilde (a{backspace} used so cursor is in the textbox and {home} gets us to the beginning of the line)
+                cy.get('#edit_textbox').type('a{backspace}{home}~');
+                cy.wait(TIMEOUTS.TINY);
+
+                // # Channel autocomplete should show
+                cy.get('#suggestionList').should('exist');
+
+                // # Write the second tilde
+                cy.get('#edit_textbox').type('~');
```

Comment:
To type second tilde making this `~~Hello1573464414914`, it should be:
```javascript
cy.get('#edit_textbox').type('{home}{rightarrow}~');
```

Reply:
Why would that be necessary? The cursor is already in place, so the result is the same. Is there any other reason I am not able to grasp?

Reply:
With your initial implementation, the next `~` is typed at the end instead of second position. 
 It became `~Hello1573643767818~` instead of expected `~~Hello1573643767818`.